EIPS/eip-888.md

```diff
@@ -0,0 +1,189 @@
+---
+eip: 888
```

Comment:
Understood. This will cause checks to fail. How can I apply for EIP number assignment? It's not clear in [EIP-1](https://eips.ethereum.org/EIPS/eip-1#eip-work-flow).